src/metabase/events/schema.clj

```diff
@@ -37,7 +36,7 @@
 
 (let [view-only      (mc/schema
                       [:map {:closed true}
-                       [:context ::view-log/context]
+                       [:context [:enum :dashboard :question :collection]]
```

Comment:
`::view-log/context` is not the right schema, since this event architecture is shared by more than view-log as a subscriber. (Also we started passing in `:collection`)

Reply:
Oops, the reason why I put `::view-log/context` here was to document the coupling of possible values of `view_log.context` to `:event/card-read`'s `context. But I didn't explain that anywhere or enforce it. I wanted to explicitly couple them because all card reads should theoretically logged in the `view_log` table now and into the future.

I've just created a PR to make sure `::view-log/context` doesn't get out of date again: [Actually validate view_log.context matches ::view-log/context on insert](https://github.com/metabase/metabase/pull/43801)